### PR TITLE
Feat/ab#75323 last update control for maps

### PR DESCRIPTION
**Pull request by MwanPygmay**
# Description

add a control to the map to know when the last modification was done. Last modification is considered as the last time the settings were SAVED, if the settings are closed it does not change.

## Useful links

[Please insert link to ticket](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/75323)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

create maps, edit settings, play around with the options.

## Screenshots

Screenshot from 2023-09-25 15-54-14

# Checklist:

( * == Mandatory ) 

- [x] * I have set myself as assignee of the pull request
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by estelafs — COMMENTED** [review]
Looks good to me and works as expected. 
@AntoineRelief  just to check if it's ok to be a map control displayed inside the map and not around the widget

**Comment by AntoineRelief** [comment]
@viniciusriquieri1 
I'll finish it, but I think you also didn't get what I meant

File: libs/shared/src/lib/components/widgets/common/display-settings/display-settings.component.html [hunk 1]
```diff
@@ -10,5 +10,36 @@ <h2>
     </ui-toggle>
   </ng-container>
   <!-- Other specific options for each widget type -->
+  <!-- === MAP === -->
+  <ng-container *ngIf="widgetType === 'map'" formGroupName="modifiedAt">
```
Comment (AntoineRelief) [inline libs/shared/src/lib/components/widgets/common/display-settings/display-settings.component.html:14]: @MwanPygmay 
@viniciusriquieri1 
@estelafs 
this one is common, and should NOT contain any logic that would check if the widget is of a specific type
we already have some examples about that
it's all the purpose of the <ng-content> below. You can inject additional elements in the component, without having to rewrite it

**Comment by AntoineRelief** [comment]
@estelafs 
I will close it because it's far from being fine
You can create another PR, in order to have a fresh start for it

See on DevOps for more info

**Pull request closed without merging**